thundra/constants.py

```diff
@@ -15,15 +15,15 @@
 LAMBDA_APPLICATION_CLASS_NAME = 'AWS-Lambda'
 LAMBDA_APPLICATION_PLATFORM = 'AWS Lambda'
 
-TRIGGER_OPERATION_NAME_TAG = 'x-thundra-trigger-operation-name'
-TRIGGER_CLASS_NAME_TAG = 'x-thundra-trigger-class-name'
-TRIGGER_DOMAIN_NAME_TAG = 'x-thundra-trigger-domain-name'
-TRIGGER_RESOURCE_NAME_TAG = 'x-thundra-resource-name'
-
-THUNDRA_TRACE_ID_KEY = 'x-thundra-trace-id'
-THUNDRA_TRANSACTION_ID_KEY = 'x-thundra-transaction-id'
-THUNDRA_SPAN_ID_KEY = 'x-thundra-span-id'
-THUNDRA_BAGGAGE_PREFIX = 'x-thundra-baggage-'
+TRIGGER_OPERATION_NAME_TAG = 'x-catchpoint-trigger-operation-name'
+TRIGGER_CLASS_NAME_TAG = 'x-catchpoint-trigger-class-name'
```

Comment:
This is not the tag, but key. So it should be `TRIGGER_CLASS_NAME_KEY`

Reply:
done